src/renderer/families/bitcoin/AccountTabs.js

```diff
@@ -0,0 +1,129 @@
+// @flow
+import React, { memo, useCallback } from "react";
+import { useTranslation } from "react-i18next";
+import styled from "styled-components";
+import type { Account, Address } from "@ledgerhq/live-common/lib/types";
+import type { ThemedComponent } from "~/renderer/styles/StyleProvider";
+
+import OperationsList from "~/renderer/components/OperationsList";
+import { SplitAddress } from "~/renderer/components/OperationsList/AddressCell";
+import Tabs from "~/renderer/components/Tabs";
+
+import TableContainer, { HeaderWrapper, TableRow } from "~/renderer/components/TableContainer";
+import Box from "~/renderer/components/Box/Box";
+import Button from "~/renderer/components/Button";
+
+import { useDispatch } from "react-redux";
+import { openModal } from "~/renderer/actions/modals";
+
+export const TableLine: ThemedComponent<{}> = styled(Box).attrs(() => ({
+  ff: "Inter|SemiBold",
+  color: "palette.text.shade60",
+  horizontal: true,
+  alignItems: "center",
+  justifyContent: "flex-start",
+  fontSize: 3,
+  flex: 1.125,
+  pr: 2,
+}))`
+  box-sizing: border-box;
+  &:last-child {
+    justify-content: flex-end;
+    flex: 0.5;
+    text-align: right;
+    white-space: nowrap;
+  }
+`;
+
+export const Cell: ThemedComponent<{ px?: number }> = styled(Box).attrs(p => ({
+  px: p.px === 0 ? p.px : p.px || 4,
+  horizontal: true,
+  alignItems: "center",
+  flex: 1.125,
+}))`
+  flex-grow: 1;
+  flex-shrink: 1;
+  display: block;
+`;
+
+type Props = {
+  account: Account,
+  parentAccount?: Account,
+};
+
+function AddressesTab({
+  freshAddresses,
+  receiveAddress,
+}: {
+  freshAddresses: Address[],
+  receiveAddress: Function,
+}) {
+  const { t } = useTranslation();
+
+  return (
+    <TableContainer>
+      <HeaderWrapper>
+        <TableLine>{t("account.address")}</TableLine>
+        <TableLine>{t("account.transaction")}</TableLine>
+      </HeaderWrapper>
+      {freshAddresses.map(({ address, derivationPath }, i) => (
+        <TableRow key={address + i}>
+          <Cell>
+            <SplitAddress color="palette.text.shade80" ff="Inter" fontSize={3} value={address} />
+          </Cell>
+          <Button
+            onClick={() => {
+              receiveAddress(address, derivationPath);
+            }}
+          >
+            {t("accounts.contextMenu.receive")}
+          </Button>
+        </TableRow>
+      ))}
+    </TableContainer>
+  );
+}
+
+function AccountTabs({ account, parentAccount }: Props) {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+
+  const freshAddresses = account.freshAddresses || [];
+
+  const receiveAddress = useCallback(
+    (address, path) => {
+      dispatch(
+        openModal("MODAL_RECEIVE", {
+          account: { ...account, freshAddress: address, freshAddressPath: path },
```

Comment:
@gre :wave: hi we need your input on this part of the code.
What we need to do is be able to trigger a receive flow from a specific bitcoin address. 
The common logic as of now doesn't allow that directly so either we do this little trick here in "emulating" an account with the address we want to receive on, or we update the receive logic so we can force these params maybe.
WDYT?

Reply:
no we shouldn't do this. we need instead to pass a second parameter that tells the index to use.
that index needs to be passed all along. typically to the bridge.receive.
we must do this properly 🙏 

in scope of this, we must make an evolution on CLI to also debug it from there, etc.. i'm sorry to not have anticipated on that. let's try to write it down for next sprint.

Reply:
ok looks like it's an index in freshAddresses? so index? but what about old addresses tho. Didn't we wanted to list used addresses too?